tensorboard/webapp/widgets/line_chart_v2/lib/utils.ts

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 
 import {Extent, Polyline, Rect} from './internal_types';
 
-export function convertRectToExtent(rect: Rect): Extent {
+function convertRectToExtent(rect: Rect): Extent {
```

Comment:
Can we continue to export these so that we can import them directly in places other than the tests?

Reply:
If code uses the functions directly then they won't be using the mocks. 

For example:

If the code you're writing does this:
```
function myFunction() {
  return convertRectToExtent(this.someRect);
}
```

But your test does this:
```
spyOn(ChartUtils, 'convertRectToExtent').and.returnValue(someExtent);
expect(myFunction()).toEqual(someExtent);
```

It won't work. Because you're replacing what ChartUtils.convertRectToExtent points to with a mock/spy but raw convertRectToExtent remains unmodified.

Reply:
Ah yea that makes sense.  Dang ok I guess we have to do it this way.